parse.go

```diff
@@ -51,6 +69,8 @@ func parseRabinString(r io.Reader, chunker string) (Splitter, error) {
 		size, err := strconv.Atoi(parts[1])
 		if err != nil {
 			return nil, err
+		} else if int(float32(size)*1.5) > BlockPayloadLimit { // FIXME - there is probably a better way to bubble up this calculation from NewRabin()
```

Comment:
@Stebalien @dirkmc I couldn't find a better way, but there got to be...

Reply:
Honestly, I'd consider enforcing this inside `NewRabin` itself (and changing the signature to return an error).

Reply:
@Stebalien this all makes sense, I prefer to do so in a new PR though, this one is becoming crowded ( too many concerns :( )

Reply:
SGTM.